server/routes/reservationsRouter.js

```diff
@@ -21,5 +29,16 @@ reservationRouter.route('/test')
     }
   })
 
+reservationRouter.route('/:garage_id')
+  .get(async (req, res) => {
+    console.log(req.params, req.query);
```

Comment:
Not sure if you meant to leave this console log in for the PR

Reply:
Gotcha, i'll remove this!